maxim/logger/pydantic_ai/client.py

```
@@ -0,0 +1,735 @@
+
+            scribe().debug(f"[MaximSDK] --- Calling: {final_op_name} ---")
+
+            try:
+                # Call the original method
+                output = original_method(self, *args, **kwargs)
+                
+                # Handle async responses with proper context preservation
+                if hasattr(output, '__await__'):
+                    scribe().debug(f"[MaximSDK] Handling coroutine for {final_op_name}")
+                    
+                    async def async_wrapper():
```

Ruff (0.12.2)

Missing return type annotation for private function `async_wrapper`

(ANN202)